client/src/containers/CategoryForm.js

```diff
@@ -33,6 +55,13 @@ class CategoryForm extends Component {
   }
 }
 
-const mapStateToProps = ({ categories }) => ({ categories });
+const mapStateToProps = ({ categories, category }) => ({
+  categories,
+  category
```

Comment:
are you guys using this somewhere in the component which i can't find ? :sweat_smile:

Reply:
Lol no we're not using it 😅